src/LibraryUtilities.ts

```diff
@@ -371,11 +371,53 @@ export function buildLibrarySectionsFromLayoutSpecs(loadedTypes: any, layoutSpec
     let miscSection = sections.find(x => x.text == "Miscellaneous");
 
     results.push(convertToDefaultSection(typeListNodes, defaultSection));
+
+    _.each(sections, function (section) {
+        if (section.text != "default" && section.text != "Miscellaneous") {
+            results.push(convertToOtherSection(typeListNodes, section));
+        }
+    })
```

Comment:
Can you help to consolidate all `default` and `Miscellaneous` strings into the same place as `libraryContainer.props.libraryController.ItemClickedEventName` and use it here? So when we change one place, everywhere else changes. Thanks!